### ISSUE TITLE
Nothing returns after `showPaymentSelector` function call

### ISSUE THREAD

**Issue by awazgyawali** [google-pay/flutter-plugin#34]
Not sure if its just me or my configuration.

I am basically checking apple pay support on iOS and once thats done, I am opening the payment dialog using `showPaymentSelector`

After successful payment I get nothing, no data in return.

```
Map<String, dynamic> data = await _pay.showPaymentSelector(
    provider: PayProvider.apply_pay,
    items:[......]
);
```

**Comment by JlUgia** [comment]
Do you have the typo in the provider name in your code too, or is it correctly spelled?: `PayProvider.apple_pay`.

**Comment by awazgyawali** [comment]
@JlUgia Sorry that's a typo here. But as this is an enum value, I am pretty sure I would have got proper error message if it was the case. 

The issue here is not about invoking apple pay dialog, its after the payment is done. I do not get `Map<String, dynamic>` value in return.

**Comment by JlUgia** [comment]
Hey @awazgyawali.
Can you share a snippet of the code that you are running?

**Comment by JlUgia** [comment]
Hey @awazgyawali, I've pushed a fix for this to version 1.0.4.
If you `flutter pub upgrade` it, you should not see the issue anymore.
Feel free to close the case once you check that the fix works for you.

**Comment by awazgyawali** [comment]
Seems fixed, I am getting values in return. Also seems another issue is fixed where shipping address was mandatory no matter what the configuration was.
